webcentral/src/templates/pages/Datenschutzhinweis.html

```diff
@@ -109,11 +142,48 @@ <h4>{% translate "Allgemeiner Datenschutzhinweis" %}</h4>
               </p>
             </li>
           </ol>
+          <h4>{% translate "Session Cookies" %}</h4>
+          <h4>{% translate "Cookies" %}</h4>
+          <p class="lead">
+            {% blocktranslate %}Auf unserer Seite sowie bei der Anzeige von Bannern setzen wir verschiedene Cookies ein, die teils nur während Ihres Besuchs gültig sind und teils darüber hinaus bestehen.
+             Dies ist aus technischen Gründen für die Bereitstellung der Funktionen und zur Gewährleistung einer korrenten Darstellung der Inhalte notwendig.
+            {% endblocktranslate %}
+          </p>
+
+          <p class="lead">
+            {% blocktranslate %}Wir setzen unter anderem folgende Cookies auf unserer Domain wissen-digital-ewb.de ein:{% endblocktranslate %}
+          </p>
+
+          <p class="lead">
+            <strong>igonesSplashWindow</strong>
+            <br />
+            {% blocktranslate %}HTTPOnly: False, Secure: False, SameSite: None, Gültigkeitsdauer: 1 Tag (bis Sitzungsende bzw. max. 1 Tag). Dieses Cookie dient dazu, bestimmte Anzeigeelemente (wie ein Splash-Fenster) während Ihres Besuchs zu steuern und ist somit nur kurzzeitig gültig.{% endblocktranslate %}
+          </p>
+
+          <p class="lead">
+            <strong>csrftoken</strong>
+            <br />
+            {% blocktranslate %}HTTPOnly: False, Secure: False, SameSite: Lax, Gültigkeitsdauer: 10 Monate. Dieses länger gültige Cookie setzt einen CSRF-Schutz-Token, um die Sicherheit Ihrer Sitzungen zu erhöhen und missbräuchliche Aktionen zu verhindern.{% endblocktranslate %}
+          </p>
+
+          <p class="lead">
+            <strong>sessionid</strong>
+            <br />
+            {% blocktranslate %}HTTPOnly: True, Secure: True, SameSite: Strict, Gültigkeitsdauer: 1 Tag. Dieses Cookie enthält eine zufällig erzeugte Session-ID zur eindeutigen Zuordnung Ihres Warenkorbs und Ihrer Bestellungen. Wird die Sitzung beendet oder das Cookie ist abgelaufen, wird der Warenkorb zurückgesetzt, sofern Sie den Bestellvorgang nicht abgeschlossen haben.{% endblocktranslate %}
```

Comment:
I think we should remove the sentence with the shppping card since we do not have a shopping card on the website? Or is that the text used on the PTJ website and should be kept?